internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -33,11 +33,15 @@ const (
 	annotationDisable      = "disable"
 	schedDomainDir         = "/proc/sys/kernel/sched_domain"
 	irqSmpAffinityProcFile = "/proc/irq/default_smp_affinity"
+	irqBannedCPUConfigFile = "/etc/sysconfig/orig_irq_banned_cpus"
 	cgroupMountPoint       = "/sys/fs/cgroup"
+	irqBalanceBannedCpus   = "IRQBALANCE_BANNED_CPUS"
 )
 
 // HighPerformanceHooks used to run additional hooks that will configure a system for the latency sensitive workloads
-type HighPerformanceHooks struct{}
+type HighPerformanceHooks struct {
+	irqBalanceConfigFile string
```

Comment:
why have this file as a member of the Hooks, when the other file locations are hardcoded?

Reply:
@haircommander This file location is platform dependent. For example, in Ubuntu, the config file is /etc/default/irqbalance. Hence making it to configure from a RuntimeConfig property. Can you check it now ?